<launch>
    <env name="SWARM_DIR" value="/home/miles/swarmpatch" />
    <node name="wifi_transmitter" pkg="transmit_wifi" type="wifi_transmitter" ns="ground" />
    <node name="command_swarm" pkg="swarm_cmd" type="command_swarm" ns="ground" />
    <node name="create_full_swarm_patch" pkg="swarmpatch" type="create_full_swarm_patch" ns="ground" />
</launch>